.github/workflows/build-images.yml

```
@@ -17,49 +17,27 @@ on:  # yamllint disable-line rule:truthy
       - 'services/**'
 
 jobs:
-  image-build-php:
+  # Deleted other jobs to keep builds nice and snappy. Restore these before
+  # merging into main branch!
+
+  # Temporary job to build an image from the PR branch. Image intended to more
+  # easily experiment with using nvm to install Node.
+  image-build-node-via-nvm:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php: [7.3, 7.4, "8.0"]
+        php: ["8.0"]
     steps:
       - name: Checkout code
         uses: actions/checkout@v2.3.4
 
-      - name: Validate docker images
-        if: ${{ github.event_name == 'pull_request' }}
-        run: docker build services/php-apache/ --build-arg PHP_VERSION=${{ matrix.php }}
-
       - name: Build and push Docker images
         uses: docker/build-push-action@v1
         if: ${{ github.event_name != 'pull_request' }}
```

@markdorison I’m opening this Draft PR with the intent to keep it open until we reach a definitive solution. I streamlined this file to only run my temporary `image-build-node-via-nvm` job and I intend to modify this line to build on pushes to this pull request. That way, I can keep this PR open and push to it to trigger new builds as needed to ease experimentation. It will publish new Docker images to a new `chromatichq/php-apache-nvm-test` image in Docker Hub. Here is the change I intend to commit next:
```suggestion
        if: ${{ github.event_name == 'pull_request' }}
```

Before I make that change, I want to make sure I get your eyes on it. The diff makes it a bit hard to follow, so [here’s a handy link to the file in this branch](https://github.com/ChromaticHQ/docker-images/blob/ce263c233949d9d8287260b3ac4d9fc288e1d349/.github/workflows/build-images.yml). Let me know if you have any questions.

---

> Before I make that change, I want to make sure I get your eyes on it.

It's not clear to me what I should be reviewing vs what is just temporary. Could we have a short call to discuss?

---

@markdorison I added this as a topic to our Eng Leads call later this afternoon, since the agenda is a bit thin anyway.

---

Reviewed during Eng leads call. As discussed, I have fixed the path to the new temporary `Dockerfile` and enabled the new workflow on pull requests.